lib/msal-node/src/client/ClientApplication.ts

```diff
@@ -106,8 +107,27 @@ export abstract class ClientApplication {
         return refreshTokenClient.acquireToken(this.initializeRequestScopes(request) as RefreshTokenRequest);
     }
 
+    /**
+     * Acquires a token silently when a user specifies the account the token is requested for.
+     *
+     * This API expects the user to provide an account object and looks into the cache to retrieve the token if present.
+     * There is also an optional "forceRefresh" boolean the user can send, to bypass the cache for access_token and id_token
+     * In case of an expired/unfound "refresh_token", an error is thrown
```

Comment:
"In case refresh token is expired or no refresh token is found,"

Reply:
addressed